hack/multi-node/bootkube-up

```diff
@@ -9,7 +9,7 @@ fi
 
 # Render assets
 if [ ! -d "cluster" ]; then
-  ../../_output/bin/${local_os}/bootkube render --asset-dir=cluster --api-servers=https://172.17.4.101:443 --etcd-servers=http://172.17.4.51:2379
+  ../../_output/bin/${local_os}/bootkube render --self-host-kubelet=false --asset-dir=cluster --api-servers=https://172.17.4.101:443 --etcd-servers=http://172.17.4.51:2379
```

Comment:
self-host-kubelet=false is the default. Do you want it here to be more easily changeable?